regen-network/regen-web
# fix: use @netlify/plugin-nextjs v4 to fix website deploy issues

## Description

**This fixes the recent issues we've been having with new deploys of our website.**

Hot fix to revert to netlify runtime v4 for web-www for now since it causes some issues with our current monorepo and bun set up.

From my discussions with the netlify support team, a potential solution for runtime v5 to work would be to move away from bun and revert to yarn/npm, which is not a light decision and should be discussed beforehand VS waiting for netlify to resolve this for bun VS moving away from netlify.



---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...


### How to test

Make sure all apps are working properly:
https://deploy-preview-2413--regen-website.netlify.app/
https://deploy-preview-2413--regen-marketplace.netlify.app/
https://deploy-preview-2413--regen-storybook.netlify.app/

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...
